seqr/views/utils/airtable_utils.py

```diff
@@ -104,3 +104,21 @@ def _populate_records(self, record_type, records, offset=None):
 
         if response_json.get('offset'):
             self._populate_records(record_type, records, offset=response_json['offset'])
+
+    def _get_samples_for_id_field(self, sample_ids, id_field, fields):
```

Comment:
copied from anvil_metdata_utils, as we now need this behavior in multiple places